packages/shared/src/components/HorizontalScroll/useScrollManagement.tsx

```diff
@@ -19,7 +19,7 @@ export const useScrollManagement = (
       const { scrollLeft, scrollWidth, clientWidth } = ref.current;
 
       setIsAtStart(scrollLeft === 0);
-      setIsAtEnd(scrollLeft + clientWidth >= scrollWidth);
+      setIsAtEnd(scrollLeft + clientWidth >= scrollWidth - 1);
```

Comment:
Why was this needed?

Reply:
Items inside was scrolling by width -1 and was not triggering the atEnd, really strange :\